_docs/overview/overview.md

```diff
@@ -0,0 +1,96 @@
+---
+title: Forseti conceptual overview
+order: 001
+hide: 
+  left_sidebar: true
+---
+
+# {{ page.title }}
+
+Forseti Security is a collection of community-driven, open-source tools to help
+you improve the security of your Google Cloud Platform (GCP) environments.
+Forseti includes core modules that you can enable, configure, and execute
+independently of each other. Community contributors can also develop add-on
+modules to offer unique capabilities. Forseti's core modules work together to
+provide their respective features, and provide a foundation that addons can
+build upon.
+
+Get started with
+[Forseti Security]({% link _docs/quickstarts/forseti-security/index.html %}).
```

Comment:
This link and every other one on this page should be corrected to point to the .md extension .html.

Reply:
Fixed, thanks, and apologies for the oversight!